### PR TITLE
DBPT-340 Use shared Docker image(s)

### PR DESCRIPTION
**Pull request by gabelton**
https://uktrade.atlassian.net/browse/DBTP-340

### CODE REVIEW

File: commands/conduit_cli.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 from commands.utils import check_aws_conn
 
+DOCKER_IMAGE_LOCATION = "public.ecr.aws/uktrade/tunnel"
```
Comment (yusufsheiqh) [inline commands/conduit_cli.py:10]:
```suggestion
CONDUIT_DOCKER_IMAGE_LOCATION = "public.ecr.aws/uktrade/tunnel"
```

File: tests/test_conduit_cli.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def test_create_task(subprocess_call):
     create_task("dbt-app", "staging", "arn::blah")
 
     subprocess_call.assert_called_once_with(
-        "copilot task run -n dbtunnel --dockerfile Dockerfile --secrets DB_SECRET=arn::blah --app dbt-app --env staging",
+        "copilot task run -n dbtunnel --image public.ecr.aws/uktrade/tunnel --secrets DB_SECRET=arn::blah --app dbt-app --env staging",
```
Comment (yusufsheiqh) [inline tests/test_conduit_cli.py:40]: I think we can make this a constant in `conftest.py`, or... is that not necessary?

We could also link the two together, so one constant used in all places.

File: commands/conduit_cli.py [hunk 3]
```diff
@@ -23,7 +25,7 @@ def get_cluster_arn(app: str, env: str) -> str:
 
 
 def create_task(app: str, env: str, secret_arn: str) -> None:
-    command = f"copilot task run -n dbtunnel --dockerfile Dockerfile --secrets DB_SECRET={secret_arn} --app {app} --env {env}"
+    command = f"copilot task run -n dbtunnel --image {DOCKER_IMAGE_LOCATION} --secrets DB_SECRET={secret_arn} --app {app} --env {env}"
```
Comment (yusufsheiqh) [inline commands/conduit_cli.py:28]:
```suggestion
    command = f"copilot task run -n dbtunnel --image {CONDUIT_DOCKER_IMAGE_LOCATION} --secrets DB_SECRET={secret_arn} --app {app} --env {env}"
```

File: tests/test_conduit_cli.py (hunk 2 above)
Comment (gabelton) [reply tests/test_conduit_cli.py:40]: I didn't want the constants linked because I'd like a change to that constant to mean that a unit test fails

Comment (yusufsheiqh) [reply tests/test_conduit_cli.py:40]: That sounds fair to me.

**Review by yusufsheiqh — APPROVED** [review]

**Review by WillGibson — APPROVED** [review]

**Pull request merged**